blueprints/simple_fpe.py

```diff
@@ -0,0 +1,31 @@
+"""
+Basic Format Preserving Encryption
+"""
+from gretel_client.transformers.transformers import SecureFpeConfig
+from gretel_client.transformers import DataPath, DataTransformPipeline
+
+xf = SecureFpeConfig(secret="2B7E151628AED2A6ABF7158809CF4F3CEF4359D8D580AA4F7F036D6F04FC6A94", radix=10,
+                     mask="1000 0000 0000 0000")
```

Comment:
Question: the whitespaces in the mask, is that just cosmetic or does have it any affect on how things work? If just cosmetic (i.e. easier to read) maybe have an in-line comment that mentions that? I notice the example card below has whitespaces init so just wanted to know if the user should line up the whitespaces in the original data and the mask.

Reply:
Yes cosmetic.. Anything that is not the mask_character get's left alone basically. I can add a comment.